code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -208,7 +208,7 @@
 
 /datum/reagent/water/holywater/reaction_mob(mob/living/M, method=TOUCH, reac_volume)
 	if(iscultist(M))
-		to_chat(M, "<span class='userdanger'>A darkness begins to spread its unholy tendrils through your mind, purging the Old God's influence!</span>")
+		to_chat(M, "<span class='userdanger'>A vile holiness begins to spread its shining tendrils through your mind, purging the Justiciar's influence!</span>")
```

Comment:
Isn't ratvar the justicar? `iscultist` now only refers to blood cult.

Reply:
i think it used to be is_servant_of_ratvar but got changed? i'll fix it

Reply:
fixed